common/src/main/java/org/jboss/hal/testsuite/testlistener/SnapshotListener.java

```diff
@@ -17,11 +17,11 @@
 import java.io.IOException;
 import java.time.LocalTime;
 
-public class TestListener extends RunListener {
+public class SnapshotListener extends RunListener {
```

Comment:
I was already thinking about `ServerStateLogger` as an alternative to `ServerStateReporter`. I personally don't like the `AfterTest` part since it's exposing details such if it will be run after testcase or atomic test and this may change.